media/css/firefox/welcome.scss

```diff
@@ -67,6 +79,12 @@
     .welcome-page3 & {
         @include text-display-md;
     }
+
+    .welcome-page4 & {
```

Comment:
Not a blocker and I don't imply you should change anything here: but I wonder if there will be a point where each welcome page would benefit from a separate stylesheet (and a base they all inherit from)?

Reply:
I wondered the same thing myself while working on this. Would this be worth an issue? I waffled about it because the changes per welcome were so minute.